editor.planx.uk/package.json

```diff
@@ -115,6 +115,7 @@
     "@testing-library/user-event": "^14.4.3",
     "@types/dompurify": "^3.0.5",
     "@types/draft-js": "^0.11.12",
+    "@types/geojson": "^7946.0.14",
```

Comment:
This is now required for v7, please see https://github.com/Turfjs/turf/blob/master/CHANGELOG.md#700